src/azul/project/hca/transformers.py

```diff
@@ -76,6 +76,7 @@ def _file_dict(f: api.File) -> MutableMapping[str, Any]:
         'indexed': f.manifest_entry.indexed,
         'name': f.manifest_entry.name,
         'sha1': f.manifest_entry.sha1,
+        'sha256': f.manifest_entry.sha256,
```

Comment:
We want to replace `sha1` with `sha256`.

Reply:
Emphasis on "replace"

Reply:
But if I remove the sha1 field the test_index_correctness test fails